images/docker-stacks-foundation/Dockerfile

```diff
@@ -117,7 +117,8 @@ RUN set -x && \
         'jupyter_core' && \
     rm -rf /tmp/bin/ && \
     # Pin major.minor version of python
-    mamba list python | grep '^python ' | tr -s ' ' | cut -d ' ' -f 1,2 >> "${CONDA_DIR}/conda-meta/pinned" && \
+    # https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-pkgs.html#preventing-packages-from-updating-pinning
```

Comment:
Future contributors will be happy to see it's an official and recommended way of pinning packages.